smart_contracts/contracts/test/key-management-thresholds/src/main.rs

```diff
@@ -64,7 +64,7 @@ pub extern "C" fn call() {
             .unwrap_or_revert();
         // Removes [43;32] key created in init stage
         account::remove_associated_key(AccountHash::new([44; 32])).unwrap_or_revert();
-        // Sets action threshodl
```

Comment:
:joy:

Reply:
What do you mean it's not spelled "hodl" ?